core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackStopRestrictionService.java

```diff
@@ -53,7 +53,7 @@ public StopRestrictionReason isInfrastructureStoppable(StackDtoDelegate stack) {
         for (InstanceGroupView instanceGroup : stack.getInstanceGroupViews()) {
             if (ephemeralVolumeChecker.instanceGroupContainsOnlyDatabaseAndEphemeralVolumes(instanceGroup)) {
                 StopRestrictionReason ephemeralResult = checkEphemeralOnlyInstanceGroupStoppable(instanceGroup, stack.getBlueprint(), cbVersion, saltCbVersion);
-                if (ephemeralResult != StopRestrictionReason.NONE) {
+                if (ephemeralResult != StopRestrictionReason.EPHEMERAL_VOLUMES && ephemeralResult != StopRestrictionReason.NONE) {
```

Comment:
This logic is incorrect as it simply just dismisses the result of checkEphemeralOnlyInstanceGroupStoppable.
With this change cluster stops could result in data loss for customers.
The checkEphemeralOnlyInstanceGroupStoppable clearly describes why each scenario result in a non-stoppable cluster.

Reply:
The real solution would be here to make sure the IMPALA service's IMPALAD role is capable of working on ephemeral volumes and survives the data loss of the volume in case of a stop-start (related Jira: https://jira.cloudera.com/browse/CDPD-34950). If this is true for this role then it can be added to the configuration and stopping for this template would be enabled. Otherwise this will not work without breaking the current functionality.